drivers/storage/portworx/component/telemetry.go

```diff
@@ -1004,41 +1008,59 @@ func getArcusRegisterProxyURL(cluster *corev1.StorageCluster) string {
 }
 
 // CanAccessArcusRegisterEndpoint checks if telemetry registration endpoint is reachable
-func CanAccessArcusRegisterEndpoint(cluster *corev1.StorageCluster) (bool, error) {
+// return true immediately if it can reach to Arcus
+// return false after failing 5 times in a row
+func CanAccessArcusRegisterEndpoint(
+	cluster *corev1.StorageCluster,
+	httpProxy string,
+) bool {
 	endpoint := getArcusRegisterProxyURL(cluster)
 	logrus.Debugf("checking whether telemetry registration endpoint %s is accessible on cluster %s",
 		endpoint, cluster.Name)
 
-	url, err := url.Parse(fmt.Sprintf("https://%s:443/auth/1.0/ping", endpoint))
-	if err != nil {
-		return false, err
-	}
-
+	url, _ := url.Parse(fmt.Sprintf("https://%s:443/auth/1.0/ping", endpoint))
```

Comment:
although this will not return error for now, from coding convention perspective, let's still handle the error, if you don't want to return it, just log if error is not nil in case it happens

Reply:
the error can be traced when sending the request, if we hit error here the url will be empty, thus when sending the request, it can be logged properly. same for parsing http proxy.

Reply:
for parsing proxy url, just looked the code of http.ProxyURL, if proxyUrl is empty it will be treated as no proxy, from coding convention perspective it's easier to track when an error is handled in place.

	// If Proxy is nil or returns a nil *URL, no proxy is used.
	Proxy func(*Request) (*url.URL, error)

Reply:
handled parsing proxy url